test/core/libp2p/multi/utils/address_converter_test.cpp

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gtest/gtest.h>
+
+#include "libp2p/multi/converters/converter_utils.hpp"
+#include "libp2p/multi/utils/protocol_list.hpp"
+extern "C" {
+#include "libp2p/multi/c-utils/protoutils.h"
+}
+using libp2p::multi::converters::addressToBytes;
+using libp2p::multi::ProtocolList;
+using kagome::common::Buffer;
+
+
+TEST(AddressConverter, Ip4AddressToBytes) {
+  auto bytes = addressToBytes(*ProtocolList::get("ip4"), "127.0.0.1").value();
+  Buffer b {127, 0, 0, 1};
+  ASSERT_EQ(bytes, "7F000001");
+
+  auto p = *ProtocolList::get("ip4");
+  struct Protocol cp;
+  cp.deccode = static_cast<int>(p.code);
+  cp.size = p.size;
+  memcpy(cp.name, p.name.data(), p.name.size());
+  char* result = (char*)malloc(256);
+  int s = 256;
+  address_string_to_bytes(&cp, "127.0.0.1", 9, &result, &s);
+  result[s] = '\0';
```

Comment:
do not use original C-utils in tests. get value from this function, write it somewhere in a variable and use it. 

then remove c-uitls